terraform/validator-sets/dev/1e5d5a74b0fd09f601ac0fca2fe7d213704e02e51943d18cf25a546b8416e9e1.node.network.keys.toml

```diff
@@ -1,4 +1,4 @@
-network_signing_private_key = "82001573a003fd3b7fd72ffb0eaf63aac62f12deb629dca72785a66268ec758b"
-network_signing_public_key = "664f6e8f36eacb1770fa879d86c2c1d0fafea145e84fa7d671ab7a011a54d509"
-network_identity_private_key = "18db36900560898178e0ad009abf1f491330dc1c246e3d6cb264f6900271d55c"
-network_identity_public_key = "b1df0ea1b4c1400454bab824e2e3ef6669e4231e2b9332020d9630fe1cfb2808"
+network_signing_private_key = "f9e1b4bd35cff88f047043aebedfbd310bb6ca762040ac51c28203fb1873e539"
```

Comment:
Why is this changing keys?

Reply:
This is an unfortunate side-effect of how we are generating the keys. Earlier we used to generate the consensus keys first, sort the nodes by peer id and then [generate network keys for them](https://github.com/libra/libra/pull/1051/commits/87b30aae3a00ad98a2804aa7bcfca2baed04de92#diff-4129faf05f43fe6140af3be6a44191adL157-L161). Now, we generate all 3 keys [in one go](https://github.com/libra/libra/pull/1051/commits/87b30aae3a00ad98a2804aa7bcfca2baed04de92#diff-4129faf05f43fe6140af3be6a44191adR129-R133) for each node, which leads to different network keys for nodes.

Reply:
@mgorven: added workaround to preserve keys.